aptos-move/move-examples/mint_nft_v2/1-Create-NFT/sources/create_nft.move

```diff
@@ -0,0 +1,187 @@
+/// This is a tutorial intended to mirror the `mint_nft` example in move-examples as closely as possible, with the only difference
```

Comment:
I think it's better to just put these instructions on aptos.dev (may be https://aptos.dev/category/nft, which would need some reorg to account for token v2) and link to that in this module. This is not the best format to read the instructions anyway

Reply:
Yeah I figured this is really not a great way to do it but was just getting the instructions out there without having to format it on aptos.dev and link to token v2 etc. I can remove these from the code and re-write and format it for the docs, or I can just link to [the current tutorial](https://aptos.dev/guides/nfts/mint-nft-cli/) or a dead link to `https://aptos.dev/guides/nfts/mint-nft-cli-v2`

Reply:
Yea that's what I'll do. We're not going to keep/maintain the original tutorial/doc page right? Can I just move this to `mint_nft` and replace the contracts and docs for anything linked to it?

Reply:
I'd prefer not replacing the existing tutorial for minting token v1. Let's keep the tutorial for token v2 completely separate (even if there's some redundancy or copy pasta)